src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int COLOR_COUNT = 11;
+
+    public String getRandomColor() {
+        int index = new Random().nextInt(COLOR_COUNT);
```

Comment:
```suggestion

public class ColorSupplier {

    public String getRandomColor() {
        int index = new Random().nextInt(Color.values().length);
```

Мы можем не использовать тут переменную COLOR_COUNT. (И если в будущем будут добавляться новые цвета нам не нужно будет ее менять)